jetty-http2/http2-http-client-transport/src/main/java/org/eclipse/jetty/http2/client/http/HttpConnectionOverHTTP2.java

```diff
@@ -74,6 +76,16 @@ public void setRecycleHttpChannels(boolean recycleHttpChannels)
         this.recycleHttpChannels = recycleHttpChannels;
     }
 
+    public int getMaxMultiplex()
+    {
+        return maxMultiplex;
```

Comment:
Avoid duplicate state:
```suggestion
        Object attachment = getAttachment();
        return (attachment instanceof ConnectionPool.Multiplexable)
            ? ((ConnectionPool.Multiplexable)attachment).getMaxMultiplex()
            : 0;
```

Reply:
@sbordet I think this could at least remove some state duplication.... but might not be needed if we can delegate all the way down to HTTP2Session.